helion/_compiler/device_function.py

```diff
@@ -420,9 +423,15 @@ def tensor_arg(
     def tensor_descriptor_arg(
         self, fake_value: torch.Tensor, block_size: list[int | torch.SymInt]
     ) -> TensorDescriptorArg:
+        import re
+
         host_function = HostFunction.current()
         block_size_expr = ", ".join(map(self.literal_expr, block_size))
+        pattern = r"triton_helpers\.div_floor_integer\(([^,]+),\s*(\d+)\)"
+        replacement = r"\1 // \2"
+        block_size_expr = re.sub(pattern, replacement, block_size_expr)
```

Comment:
I believe this should unify the triton_helpers handling: https://gist.github.com/yf225/dbb4045c44df97c902906290e0f6affa